socioeconomic-data/data-sources.md

```diff
@@ -29,81 +29,47 @@ New content is announced on the [home page](/) and [news page](/about/news), so
+- **Methodologies**: Social sciences methodologies are quite diverse and complex as they try to investigate complex, changeable, and often intangible social phenomena such as trust, ideology, or socioeconomic status. Thus, social sciences use both quantitative and qualitative research methods, often in combination, ranging from large scale quantitative surveys, to interviews or ethnographic research. As such, socioeconomic research methodologies and data often differs significantly from the those typically used in the life sciences. 
+- **Sensitive data** and privacy considerations: due to the prevalence of sensitive and personal data in the domain of socioeconomic research, access to such data is often limited by serious data protection and privacy concerns, often making the use and sharing of such data difficult. Solutions such as pseudonimising data, or providing access via secure environments such as the [ODDISEI Secure ANalysis Environment (SANE)](https://odissei-data.nl/en/2022/02/secure-analysis-environment-sane-secure-data-for-social-sciences-and-humanities/) can often provide some level of access to sensite data, yet much of this data is only available on request, or as aggregated data. 
 
 ### Existing approaches
 
-Mention existing appreaches related to the topic.
+The pandemic of COVID-19 has shown the impact of socioeconomic factors in terms of governance and state-citizens relations when it comes to an efficient response of science (to deal with infectious diseases) for the benefit of the society. 
 
-## Standards, schemas, ontologies and vocabularies 
+Indicatively, there has been a considerable number of social  surveys dealing with the pandemic and its consequences in several aspects of the everyday life of citizens at global and EU level. There has been, compared to previous pandemics, a major difference, meaning that the progress of information, digitisation, statistics included, citizens were aware of the extent, the seriousness of the various whilst at the same time citizens received information about scientific efforts and achievements to deal with diseases. 
 
-<!--- Example of topic, optional --->
+From the demography perspective, there have been surveys and related publications proving the relation of household structures, housing conditions (Giorgi & Boertien 2021), the role of nursing homes, especially during the first wave (Bernandi et all 2021), the role of gender or socioeconomic status (Horton 2021) etc. From the perspective of political science, psychology or sociology sciences’  perspective, surveys that studied the effect of conspiracy theories regarding vaccines, propagation of virus and others, have show clusters of people that relate resistance to vaccines with respect to personal ideology , adherence to conspiracy theories or that lower level of education, religiosity, conspiracy thinking can be an indicator for vaccine resistance (Haaksonen & Furman 2022). 
```

Comment:
I think we need to find a way to add these references as it has to be consistent from other pages.
Could you find a DOI for all of them so we can add them maybe?